src/wazuh_db/wdb.h

```diff
@@ -1763,6 +1765,16 @@ int wdb_parse_task_delete_old(wdb_t* wdb, const cJSON *parameters, char* output)
  */
  int wdb_parse_agents_clear_vuln_cve(wdb_t* wdb, char* output);
 
+/**
+ * @brief Function to parse the vuln_cve update action.
+ * 
+ * @param [in] wdb The global struct database.
+ * @param [in] input String with the the data in json format.
+ * @param [out] output Response of the query.
+ *  * @return 0 Success: response contains "ok".
+ *        -1 On error: response contains "err" and an error description.
+ */
+ int wdb_parse_agents_update_status_vuln_cve(wdb_t* wdb, char* input, char* output);
```

Comment:
Would it be possible to follow the same name convention ?
The helper method has another word order`
wdb_agents_vuln_cve_update_status()`

Reply:
Yes, we have mixed the name order in several functions. And there're another more obvious examples here.

wdb_agents_insert_vuln_cve (wdb_agents.c)
wdb_agents_vuln_cve_insert (wdb_agents_helper.c)
wdb_agents_clear_vuln_cve (wdb_agents.c)
wdb_agents_vuln_cve_clear (wdb_agents_helper.c) 

I'll take a look.

Reply:
I made the change you requested!